pkgs/applications/misc/bibletime/default.nix

```diff
@@ -1,30 +1,45 @@
 { lib, mkDerivation, fetchurl, cmake, pkg-config, sword, boost, clucene_core
-, qtbase, qttools, qtsvg, qtwebkit
-}:
+, qtbase, qttools, qtsvg, qtwebkit, perlPackages, docbook_xml_dtd_45
+, docbook_xsl_ns }:
 
 mkDerivation rec {
 
-  version = "2.11.2";
+  version = "3.0";
 
   pname = "bibletime";
 
   src = fetchurl {
-    url = "mirror://sourceforge/bibletime/${pname}-${version}.tar.xz";
-    sha256 = "1s5bvmwbz1gyp3ml8sghpc00h8nhdvx2iyq96iri30kwx1y1jy6i";
+    url =
+      "https://github.com/bibletime/bibletime/releases/download/v${version}/${pname}-${version}.tar.xz";
+    sha256 = "08i6nb9a7z0jpsq76q0kr62hw6ph9chqjpjcvkimbcj4mmifzgnn";
   };
 
   nativeBuildInputs = [ cmake pkg-config ];
   buildInputs = [
-   sword boost clucene_core
-   qtbase qttools qtsvg qtwebkit
- ];
-
-  preConfigure =  ''
+    sword
+    boost
+    clucene_core
+    qtbase
+    qttools
+    qtsvg
+    qtwebkit
+    perlPackages.Po4a
+    docbook_xml_dtd_45
+  ];
+
+  preConfigure = ''
     export CLUCENE_HOME=${clucene_core};
     export SWORD_HOME=${sword};
   '';
 
-  cmakeFlags = [ "-DUSE_QT_WEBKIT=ON" "-DCMAKE_BUILD_TYPE=Debug" ];
+  cmakeFlags = [
+    "-DUSE_QT_WEBKIT=ON"
```

Comment:
This options seems to be long gone
https://github.com/bibletime/bibletime/commit/6e3a67ebb9f8953754737e3aafb009ae9b3a307f
Does this compile without qtwebkit?

Reply:
@veprbl Removed